### DISCUSSION TITLE
Support for conversion of whole lists at once, or to 'nudge' conversion into specific UTM zone

### DISCUSSION THREAD

**Discussion by pyotyr** [Turbo87/utm#101] [Ideas]
I use your library in most of the code I write. I usually convert over 50 coordinates at once. Usually from LatLon first to UTM then back again to LatLon to save. 

I have a problem: when the LatLons are on a border between 2 UTM zones, some of the coordinates are in one zone and the others are in a different zone. If I could pass a whole list or array at once, then the code could output everything in the same UTM zone. Alternatively, when converting a single point at a time you could pass a parameter for what UTM zone you would prefer it to output in.

I'm down to modify the library myself but I have never modified libraries before so I'd like to figure out the best way to go about it.

Thanks!

**Comment by bartvanandel** [comment]
Forcing points into a specific zone has been possible for a while now, using `force_*` parameters. See code for details. The upcoming 0.8.0 release (see #129) will include some fixes to make it work even better.

Also, if you feed a list of points (actually list of lat, list of lon; requires `numpy`), all points will be put into the same zone already. By default, the zone will be determined by the first point in the list, but if you know your points will be in multiple zones, best to tell `utm` what zone to use using the `force_*` parameters.
